src/Http/Routing/src/Builder/MapActionEndpointRouteBuilderExtensions.cs

```diff
@@ -15,6 +16,12 @@ namespace Microsoft.AspNetCore.Builder
     /// </summary>
     public static class MapActionEndpointRouteBuilderExtensions
     {
+        // Avoid creating a new array every call
+        private static readonly string[] GetVerb = new[] { "GET" };
```

Comment:
I think there are static readonly fields somewhere with these strings. Using them is ever so slightly faster during route matching as they match on reference.

Reply:
I looked and couldn't find anything like that. I must have been thinking of something else.

Reply:
You might be thinking of 

https://github.com/dotnet/aspnetcore/blob/5bf4272bd6e8bd7901b2228081ce9896f4ad3ba7/src/Http/Routing/src/Builder/EndpointRouteBuilderExtensions.cs#L19-L23

Same project. We could make those internal and reference those to save allocating the 5 extra arrays and strings. Maybe we should just merge the classes.